tests/test_data_models/test_notebook.py

```diff
@@ -131,10 +132,13 @@ def test_cells_sub(self) -> None:
 
         assert type(dl1) == type(dl2) == Cells[Cell]
         assert type(diff) == Cells[Tuple[List[Cell], List[Cell]]]
-        assert diff == Cells(  # type: ignore
+
+        expected = Cells[CellsPair](  # type: ignore
```

Comment:
Do we still need the `# type: ignore` here?

Reply:
I'll check

Reply:
Noice! Can we drop this then?